lib/morph/docker_runner.rb

```diff
@@ -67,6 +60,25 @@ def self.compile_and_start_run(
       }
 
       c = Docker::Container.create(container_options)
+
+      Dir.mktmpdir('morph') do |dest|
+        copy_config_to_directory(repo_path, dest, false)
+        yield(:internalout, "Injecting scraper and running...\n")
+
+        # Rather than using archive_in we're doing this more roundabout way
+        # because archive_in seems to have very broken handling of directories
+        # TODO Submit a fix to the docker-api gem to fix this
+        # In the meantime get something more long-winded working here
+
+        tar_file = Docker::Util.create_dir_tar(dest).path
+        File.open(tar_file, "rb") do |tar|
+          c.archive_in_stream("/app") do
+            tar.read(Excon.defaults[:chunk_size]).to_s
+          end
+        end
+        File.unlink(tar_file)
```

Comment:
How about changing this method to `.delete`? I had to look "unlink" up because I wasn't sure what it did whereas I definitely would have known what "delete" does.